pkgs/development/compilers/cudatoolkit/redist/extension.nix

```diff
@@ -11,7 +11,7 @@ final: prev: let
     "11.4" = ./manifests/redistrib_11.4.4.json;
     "11.5" = ./manifests/redistrib_11.5.2.json;
     "11.6" = ./manifests/redistrib_11.6.2.json;
-    "11.7" = ./manifests/redistrib_11.7.0.json;
```

Comment:
I'm confused whether we're doing the 11.7.1 upgrade in this PR or not... It appears as though the upgrade is still being done here but not in `pkgs/development/compilers/cudatoolkit/versions.toml`? Does `./manifests/redistrib_11.7.1.json` exist?